### PR TITLE
Get/Set/Delete Notification Config

### PR DESCRIPTION
**Pull request by vadmeste**
Still some few changes needed before merging, do not review

### CODE REVIEW

**Comment by vadmeste** [comment]
@krishnasrinivas, could you take a look at this patch ? especially how {set,get,delete}-bucket-notification.js are calling and importing our APIs.

**Comment by harshavardhana** [comment]
> @krishnasrinivas, could you take a look at this patch ? especially how {set,get,delete}-bucket-notification.js are calling and importing our APIs.

We need to add some test cases as well white we are at it.

**Comment by krishnasrinivas** [comment]
@vadmeste will look

File: src/test/unit/test.js [hunk 1]
```diff
@@ -876,59 +869,9 @@ describe('Client', function() {
       })
     })
 
-
-    describe('#deleteBucketNotification()', () => {
-      it('remove all bucket notifications', (done) => {
```
Comment (harshavardhana) [inline src/test/unit/test.js]: test case removed?

Comment (vadmeste) [inline src/test/unit/test.js]: well, all mock test cases will be removed later right?

**Comment by harshavardhana** [comment]
> well, all mock test cases will be removed later right?

If its not added here we should update the functional-test.js

**Comment by harshavardhana** [comment]
This is still a WIP?

**Comment by vadmeste** [comment]
No.

**Pull request merged**